app/components/service/includes/resources.html

```diff
@@ -6,31 +6,41 @@
       <table class="table">
         <tbody>
           <tr>
-            <td>CPU limits</td>
-            <td ng-if="service.LimitNanoCPUs">
-              {{ service.LimitNanoCPUs / 1000000000 }}
+            <td>CPU limit<portainer-tooltip position="top" message="Maximum number of CPU the service can use (0 to unlimited)."></portainer-tooltip></td>
+            <td>
+              <input class="input-sm" type="number" step="0.1" ng-model="service.LimitNanoCPUs" ng-change="updateServiceAttribute(service, 'LimitNanoCPUs')" ng-disabled="isUpdating"/>
             </td>
-            <td ng-if="!service.LimitNanoCPUs">None</td>
-          </tr>
-          <tr>
-            <td>Memory limits</td>
-            <td ng-if="service.LimitMemoryBytes">{{service.LimitMemoryBytes|humansize}}</td>
-            <td ng-if="!service.LimitMemoryBytes">None</td>
-          </tr>
-          <tr>
-            <td>CPU reservation</td>
-            <td ng-if="service.ReservationNanoCPUs">
-              {{service.ReservationNanoCPUs / 1000000000}}
+            <td>CPU res<portainer-tooltip position="top" message="Minimum number of CPU guaranteed to the service."></portainer-tooltip></td>
+            <td>
+              <input class="input-sm" type="number" step="0.1" ng-model="service.ReservationNanoCPUs" ng-change="updateServiceAttribute(service, 'ReservationNanoCPUs')" ng-disabled="isUpdating"/>
             </td>
-            <td ng-if="!service.ReservationNanoCPUs">None</td>
           </tr>
           <tr>
-            <td>Memory reservation</td>
-            <td ng-if="service.ReservationMemoryBytes">{{service.ReservationMemoryBytes|humansize}}</td>
-            <td ng-if="!service.ReservationMemoryBytes">None</td>
+            <td>Mem limit in MB<portainer-tooltip position="top" message="Maximum memory the service can use (0 to unlimited)."></portainer-tooltip></td>
```

Comment:
Please replace the `Mem limit in MB` label text with `Memory limit (MB)`.

Also, please replace the tooltip message with: `Maximum memory usage per task (set to 0 for unlimited)`